CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# Contributing to CRA Template Truemark
+
+Thank you for your interest in contributing to our project! We appreciate your interest in contributing.
+
+## How to Contribute
+
+- Fork the Template: On GitHub, click "Fork" to create your own copy.
+- Create a issue first if you are adding fixes for the template.
+- Clone the forked repository to your local machine.
+
+    ```md 
+    git clone https://github.com/TruemarkDev/cra-template-truemark.git for https.
+
+    git clone git@github.com:TruemarkDev/cra-template-truemark.git for ssh.
+    ```
+
+- Install dependencies using `npm install`.
+- Start the project using `npm start`.
+- Check out to a new branch.
+- Make your changes and commit them.
+- Push your changes to repository.
+- Create a pull request with a clear title and description.
+
+## Code Guidelines
+
+- Follow consistent coding style.
+- Write clear and meaningful commit messages.
```

Comment:
We might want to point them to specific section about writing git commit messages in our  handbook for this.